trust/module.c

```diff
@@ -287,6 +287,11 @@ parse_argument (char *arg,
 		free (gl.paths);
 		gl.paths = value ? strdup (value) : NULL;
 
+	} else if (strcmp (arg, "verbose") == 0) {
+		if (strcmp (value, "yes") == 0)
+			p11_message_loud ();
+		else if (strcmp (value, "no") == 0)
```

Comment:
Wouldn't it have more sense to call `p11_message_quiet` in every other case other that `value` being `"yes"`? Or is it guaranteed somewhere that the values could only be `"yes"` or `"no"`?

Reply:
There is technically no reason to not accept other values, but I think it would be good to be consistent with  [`_p11_conf_parse_boolean()`](https://github.com/p11-glue/p11-kit/blob/master/p11-kit/conf.c#L500) which considers other values invalid.